forem/forem
# [WIP] Add scroll to Anchor

##  What type of PR is this? (check all applicable)


## Description
Anchors are not currently caught by the router

## Related Tickets & Documents
#10234

## QA Instructions, Screenshots, Recordings
- Go to [this post](https://dev.to/eduherminio/waiting-in-arduino-ii-millis-function-to-the-rescue-33k6#going-back-to-our-case-study)
- Click on `case study` link
- See anchor be picked up

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://thumbs.gfycat.com/ActualFlawedDesertpupfish-small.gif)
